packages/iov-bns/src/util.ts

```diff
@@ -38,8 +38,17 @@ export function encodeBnsAddress(prefix: "iov" | "tiov", bytes: Uint8Array): Add
 }
 
 /** Decodes a printable address into bech32 object */
-export function decodeBnsAddress(address: Address): { readonly prefix: string; readonly data: Uint8Array } {
-  return Bech32.decode(address);
+export function decodeBnsAddress(
+  address: Address,
+): { readonly prefix: "iov" | "tiov"; readonly data: Uint8Array } {
```

Comment:
Is it time for an `IovBech32Prefix` type?

Reply:
Maybe, I don't know. Not too urgent in my opinion.

Reply:
Not urgent but we awkwardly repeat `"iov" | "tiov"` in a lot of places.

Reply:
okay okay, you get it :)